### DISCUSSION TITLE
descente axe z a chaque passe

### DISCUSSION THREAD

**Discussion by yaquesfr** [arkypita/LaserGRBL#2271] [Ideas]
bonjour

serait il possible de rajouter la possibilité de descendre l'axe z a chaque passe pour la découpe

**Comment by Tribo-2dot0** [comment]
> serait il possible de rajouter la possibilité de descendre l'axe z a chaque passe pour la découpe

Menu Grbl > Settings... aba GCode > Multiple Passes
